docs/sphinx/_toc.yml.in

```diff
@@ -61,11 +57,6 @@ subtrees:
   - url: https://github.com/amd/rocm-examples
     title: GitHub examples
```

Comment:
Should we consider calling it ROCm examples, instead of GitHub examples? Or ROCm examples in GitHub?

Reply:
I think it's clear as-is. I'm not sure examples of GitHub is a thing...and even if it was, it would be very weird to include that on the main ROCm docs site.

Reply:
GitHub examples seems to imply examples for GitHub, not ROCm.

Reply:
Sounds good. Should we check how AECG has documented their examples?

Reply:
Updated the provide section